src/main/java/entities/CollaborativeTask.java

```diff
@@ -0,0 +1,261 @@
+package entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class CollaborativeTask extends Task implements Timeblockable{
+    private boolean recurring;
+    private String frequency;
+    private ArrayList<LocalDateTime> timeBlocks;
```

Comment:
Sorry I just have one more thing. Thanks for making the changes I requested but for timeBlocks, I requested to have the type as **ArrayList < ArrayList < LocalDateTime > >** instead of ArrayList < LocalDateTime > . Sorry but I think this has to be changed as well as the getter and setter you made for timeBlocks.